cmd/dashboard/root.go

```diff
@@ -1,31 +1,73 @@
-// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+// See Copyright Notice in LICENSE
 //
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
 package dashboard
 
 import (
-	"github.com/tani-yu/dogleash/cmd"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/tani-yu/dogleash/cmd"
+	datadog "gopkg.in/zorkian/go-datadog-api.v2"
 )
 
+// MyDashboards widget form dashboard struct
+type MyDashboards struct {
+	MyDashboards []datadog.Board `json:"dashboards,omitempty"`
+}
+
 // dashboardCmd represents the dashboardCmd command
 var dashboardCmd = &cobra.Command{
 	Use:   "dashboard",
 	Short: "Perform operations related to dashboard of Datadog",
 }
 
+// getJSONDataFromAPI **WORKAROUND** get JSON from dashboard api
+func getJSONDataFromAPI(path string) []byte {
+	baseurl := "https://api.datadoghq.com/api/v1/"
+	param := "?api_key=" + viper.GetString("api_key") + "&application_key=" + viper.GetString("app_key")
```

Comment:
Don't compose a query parameter in hand. Use [url](https://golang.org/pkg/net/url/) package instead.

See: https://stackoverflow.com/a/26987017